data/rpm/flameshot.spec

```diff
@@ -1,5 +1,5 @@
 Name: flameshot
-Version: 0.8.0
+Version: 0.8.1
 Release: 1%{?dist}
 Summary: Powerful yet simple to use screenshot software
 Summary(eu-ES): Potente pero simple de usar software de capturas
```

Comment:
The changelog in the spec file was not updated.

Reply:
Do the rpm files need to be re uploaded or is it good enough to just fix on the main branch?

Reply:
No need to re-upload, it's just that the changlog needs to be updated.